ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -57,11 +57,23 @@ open class TwoLineTitleView: UIView {
         case dark
     }
 
+    @objc(MSFTwoLineTitleViewAlignment)
+    public enum Alignment: Int {
+        case center
+        case leading
+    }
+
     @objc(MSFTwoLineTitleViewInteractivePart)
     public enum InteractivePart: Int {
-        case none
-        case title
-        case subtitle
+        case none = 0
+        case title = 1
+        case subtitle = 2
+        case all = 3
```

Comment:
Any reason for the explicit enum integers? Is this meant to be a bit mask?

Reply:
Yes, this is supposed to be a bit mask. I wanted to use `OptionSet`, but that doesn't appear to play well with Objective C, so I figure this might be the next best thing.